src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -1585,54 +1586,54 @@ public boolean test(SCMTraitDescriptor<?> scmTraitDescriptor) {
 
             IconSet.icons.addIcon(
                     new Icon("icon-github-logo icon-sm",
-                            "plugin/github-branch-source/images/16x16/github-logo.png",
-                            Icon.ICON_SMALL_STYLE));
+                            "plugin/github-branch-source/images/svgs/sprite-github.svg#github-logo",
```

Comment:
I've left the old github-logo image file in as it's used here: https://github.com/jenkinsci/github-branch-source-plugin/blob/42a2234b2a501f299f10206bad81c2ff9a87cd8d/src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubOrgMetadataAction.java#L82

I'm not aware of a way to trigger that code though

Reply:
Could we return the same SVG regardless of `size` value?

Reply:
Yes we can, I'm just wary of touching code I can't test.

My point was I don't know if this code is possible to hit, at least on github.com